lib/prm/msp_api/api.ex

```diff
@@ -0,0 +1,57 @@
+defmodule PRM.MSPAPI do
+  @moduledoc """
+  The boundary for the MSPAPI system.
+  """
+
+  import Ecto.{Query, Changeset}, warn: false
+  alias PRM.Repo
+
+  alias PRM.MSP
```

Comment:
aliases should be grouped [together](https://github.com/christopheradams/elixir_style_guide#module-attribute-ordering)